hotkeys.py

```diff
@@ -58,37 +52,34 @@ def get_context(app):
         result = applescript.tell.app("iTerm2", "return tty of current session of current tab of front window")
         tty = basename(result.out)
         cmd = ["ps", "-t", tty, "-opid=,stat=,command="]
-        proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True)
-        out, err = proc.communicate()
+        out = subprocess.run(cmd, capture_output=True, universal_newlines=True, check=True).stdout
         context = "🤷🏻‍♂️"
-        if proc.returncode == 0:
-            process_lines = out.split("\n")
-            # S+ or R+
-            # TODO parse the process line, then check for the flag
-            processes = [p for p in process_lines if "S+" in p]
-            if len(processes) == 0:
-                processes = [p for p in process_lines if "R+" in p]
+        process_lines = out.split("\n")
+        # S+ or R+
```

Comment:
After reading the `man` page for `ps`, I think we might just be able to parse for `+` to find the process we are looking for 🤔

Reply:
I think in some cases, I saw two `S+` processes.

Using a `make` command that performs other commands generates multiple `S+` processes.
I was originally using `+` and got `S+` and `R+` at one point.

Reply:
Here's one example, using `aws ec2 help`:
```
Found an unexpected number of processes [
'82125 S+   /usr/local/Cellar/python@3.9/3.9.2_1/Frameworks/Python.framework/Versions/3.9/Resources/Python.app/Contents/MacOS/Python /usr/local/bin/aws ec2 help',
'82131 S+   less -R'
]
```

Reply:
Maybe we should split this discussion into an issue.

Reply:
Created #2